src/Service/TagsService.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+
+namespace ShipEngine\Service;
+
+use ShipEngine\Model\Tags\Tag;
+use ShipEngine\ShipEngineError;
+
+/*
+ * Service to create tags
+ */
+
+final class TagsService extends AbstractService
+{
+    private const CREATE = 'create_tag';
```

Comment:
Probably don't need this here. It is private to the class so unless it is being used multiple times in the class (which it isn't) then it is just misdirection.